spec/fixtures/licenses-dir-with-multiple-license-files/LICENSES/MIT.txt

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Are there corresponding tests for these fixtues/

Reply:
I think so, I added this to the `spec/fixtures/fixtures.yml`, which I believe is tested by `spec/fixture_spec.rb`. It's a fairly basic test though. Is there another test that lists out all the found licenses, or should I extend `spec/fixture_spec.rb` to support it?